Rollback telegram token as well

```
Caused by: org.postgresql.util.PSQLException: ERROR: update or delete on table "users" violates foreign key constraint "fk_token_on_user" on table "tokens"
  Detail: Key (id)=(49) is still referenced from table "tokens".
```

--- a/src/main/java/io/kyberorg/yalsee/services/user/UserOperationsService.java
+++ b/src/main/java/io/kyberorg/yalsee/services/user/UserOperationsService.java
@@ -131,6 +131,7 @@ public class UserOperationsService {
         OperationResult createTelegramTokenResult = tokenService.createTelegramConfirmationToken(createdUser);
         if (createTelegramTokenResult.ok()) {
             telegramConfirmationToken = createTelegramTokenResult.getPayload(Token.class);
+            rollbackTasks.push(RollbackTask.create(Token.class, telegramConfirmationToken));
         } else {
             telegramConfirmationToken = null;
             log.warn("{} failed to create Telegram token for {}. OpResult: {}",

